CMakeLists.txt

```diff
@@ -1244,17 +1244,7 @@ install(
 
 
 # Font files
-if(UNIX AND NOT APPLE)
-  # Ubuntu and OpenSans are already installed by package manager
-  install(
-    DIRECTORY
-      "${CMAKE_CURRENT_SOURCE_DIR}/res/fonts"
-    DESTINATION
-      "${MIXXX_INSTALL_DATADIR}"
-    PATTERN "OpenSans*" EXCLUDE
-    PATTERN "Ubuntu*" EXCLUDE
-  )
-else()
+if(NOT UNIX OR APPLE)
```

Comment:
This is confusing to me. How about `if(WIN32 OR APPLE)`?

Reply:
What if someone ports mixxx to some other OS non-unix OS? :D

How about `if(APPLE OR NOT UNIX)`?

Reply:
Every once in a while we get a pull request to fix some BSD-specific thing. I don't know if installing these fonts would be necessary for them.

Reply:
@rrrapha could you help us out here? Should we install these fonts on BSD?

Reply:
No:

https://www.freebsd.org/cgi/ports.cgi?query=open-sans&stype=all
https://www.freebsd.org/cgi/ports.cgi?query=ubuntu-font&stype=all

Reply:
Okay, so let's do `if(WIN32 OR APPLE)`

Reply:
Do you find `if(APPLE OR NOT UNIX)` harder to understand? We need to look out for all the DJs on Haiku/BeOS ;-)

Reply:
If anyone wanted to do that, first they'd have to port all of Mixxx's dependencies. I think it's more likely that someone would try to run Mixxx on Redox.

Reply:
> Do you find if(APPLE OR NOT UNIX) harder to understand?

Yes. `if(APPLE OR WIN32)` is easier to understand.